files/image_config/corefile_uploader/core_uploader.service

```diff
@@ -1,11 +1,13 @@
 [Unit]
 Description=Host core file uploader daemon
-Requires=updategraph.service
-After=updategraph.service
+Requires=syslog.service
+After=syslog.service
 
 [Service]
 Type=simple
 ExecStart=/usr/bin/core_uploader.py
+StandardOutput=null
+Restart=on-failure
```

Comment:
we need back off here? if core_uploader is constantly restarting?

Reply:
No need. The only failure that crashes the service, is some fatal things like, sonic_version.yaml does not have expected attributes or some fatal system related failure (which should crash many more ...), and running out of disk space.

The one thing, I would need to take care of "running out of disk space". This I can take care of inside the script.